Oqtane.Shared/Shared/PolicyNames.cs

```diff
@@ -1,10 +1,19 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Text;
 
-namespace Oqtane.Shared {
-    public class PolicyNames {
+namespace Oqtane.Shared
+{
+    public class PolicyNames
```

Comment:
If @sbwalker agree with this change I'd like to use `static readonly` field instead of `const`

Reply:
You can't use ```static readonly``` in attributes though.  Ie.:

```
        [HttpGet("{id}")]
        [Authorize(Policy = PolicyNames.ViewModule)]
        public List<HtmlTextInfo> Get(int id)
        {
            var list = new List<HtmlTextInfo>();
            try
            {
                HtmlTextInfo htmlText = null;
                if (_entityId == id)
                {
                    htmlText = _htmlText.GetHtmlText(id);
                    list.Add(htmlText);
                }
            }
            catch (Exception ex)
            {
                _logger.Log(LogLevel.Error, this, LogFunction.Read, ex, "Get Error {Error}", ex.Message);
                throw;
            }
            return list;
        }
```

Reply:
You are right I didn't notice they are using there